org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/testing/signature/fixture/SignatureSubFixtureDo.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2010-2022 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+package org.eclipse.scout.rt.dataobject.testing.signature.fixture;
+
+import javax.annotation.Generated;
+
+import org.eclipse.scout.rt.dataobject.DoEntity;
+import org.eclipse.scout.rt.dataobject.DoValue;
+import org.eclipse.scout.rt.dataobject.TypeName;
+import org.eclipse.scout.rt.dataobject.TypeVersion;
+import org.eclipse.scout.rt.dataobject.fixture.DataObjectFixtureTypeVersions;
+
+@TypeName("scout.SignatureSubFixture")
+@TypeVersion(DataObjectFixtureTypeVersions.DataObjectFixture_1_0_0.class)
```

Comment:
static import
Use version DataObjectFixture_1_0_0_034 just as an additional test (and change in expected)?

Reply:
Done